app/policies/document_policy.rb

```diff
@@ -14,4 +14,28 @@ def publish?
   end
 
   alias_method :unpublish?, :publish?
+
+  def pre_production_formats
+    Rails.env.development? ? [] : PRE_PRODUCTION
+  end
+
+  def removed_pre_production?
+    !pre_production_formats.include?(document_class.document_type)
+  end
```

Comment:
I've thought about this a bit, perhaps something like:

```ruby
def environment_restricted_formats
  Rails.env.development? ? [] : PRE_PRODUCTION
end

def restricted_by_environment?
  environment_restricted_formats.include?(document_class.document_type)
end

def user_organisation_owns_document_type
  ... && !restricted_by_environment?
end

def gds_editor
   ... && !restricted_by_environment?
end
```

I find those names a bit more intuitive, but keen to hear other ideas. What do you think?